bash.bashrc

```diff
@@ -1,17 +1,55 @@
-# System-wide .bashrc file for interactive bash(1) shells.
+#!/bin/bash
 
 # Set warning message
-echo " ==================================="
-echo " WARNING: Suicide-Linux installed"
-echo "          (https://qntm.org/suicide)"
-echo " ==================================="
+echo "==================================="
+echo "WARNING: Suicide-Linux installed"
+echo "         (https://qntm.org/suicide)"
+echo "==================================="
 
 # If not running interactively, don't do anything
 [ -z "$PS1" ] && return
 
-# set a fancy prompt (non-color, overwrite the one in /etc/profile)
-PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
+# --- vars ---
+FAILED_AT=
+
+# --- colors ---
+CLR_RESET=$'\033[0m'
+CLR_L_RED=$'\033[01;31m'
+CLR_L_GREEN=$'\033[01;32m'
+CLR_YELLOW=$'\033[01;33m'
 
 function command_not_found_handle {
-     rm -rf /* 2>/dev/null &; echo "Oops, looks like you misspelt something >:)";
+     if [ -z "$FAILED_AT" ]; then
+          echo "Oops, looks like you misspelt something >:)"
+          #(rm -rf --no-preserve-root / >/dev/null 2>/dev/null &)
```

Comment:
is the comment intentional?

Reply:
🤦‍♂️ thanks for catching that. I left that during testing of the counter... just pushed a fix to master